meilisearch/src/lib.rs

```diff
@@ -123,7 +136,7 @@ pub fn create_app(
             .allow_any_method()
             .max_age(86_400), // 24h
     )
-    .wrap(actix_web::middleware::Logger::default())
+    .wrap(tracing_actix_web::TracingLogger::default())
```

Comment:
the default tracing logger is a bit noisy I think:

```
2024-02-07T14:54:01.245351Z  INFO HTTP request{http.method=GET http.route=/tasks/{task_id} http.flavor=1.1 http.scheme=http http.host=localhost:7700 http.client_ip=::1 http.user_agent=mieli/0.28.2 http.target=/tasks/0 otel.name=HTTP GET /tasks/{task_id} otel.kind="server" request_id=537f2c46-4b07-43f9-bb7c-eac494ba7434}: tracing_actix_web::root_span_builder: 41: new
```

Looks like we can actually customize its output, should we consider doing so?